### PR TITLE
Adding yarn option

### PR DESCRIPTION
**Pull request by bettercalldough**
Some developers may wish to use Yarn instead of npm, so I have added yarn alternative scripts.

### CODE REVIEW

**Comment by bettercalldough** [comment]
I signed it @googlebot

**Comment by bettercalldough** [comment]
Thanks @leo6104 have fixed the typo.

File: guides/getting-started.md [hunk 1]
```diff
@@ -5,29 +5,47 @@ For existing apps, follow these steps to begin using Angular Material.
...
 install the `@angular/animations` module and include the `BrowserAnimationsModule` in your app.
 
+#### NPM
 ```bash
 npm install --save @angular/animations
 ```
 
+#### Yarn
+```bash
+yarn install @angular/animationa
```
Comment (julianobrasil) [inline guides/getting-started.md]: typo: animationa => animations

File: guides/getting-started.md [hunk 2]
```diff
@@ -5,29 +5,47 @@ For existing apps, follow these steps to begin using Angular Material.
...
 
+#### NPM
 ```bash
 npm install --save @angular/material @angular/cdk
 ```
+#### Yarn
+```bash
+yarn add @angular/material @angular/cdk
+```
+
```
Comment (jelbourn) [inline guides/getting-started.md]: I think this needs something that clearly conveys it would be one or the other, not both.

Comment (bettercalldough) [reply guides/getting-started.md]: @jelbourn something like `We have added install scripts so you can pick between **npm** or **yarn**`

Comment (jelbourn) [reply guides/getting-started.md]: Right under "Step 1", it should say 

"You can use either the `npm` or `yarn` command-line tool to install packages. Use whichever is appropriate for your project in the examples below."

Comment (bettercalldough) [reply guides/getting-started.md]: @jelbourn added it 👍

**Comment by bettercalldough** [comment]
@jelbourn seems Travis didn't like it possibly because of the file being updated

**Comment by jelbourn** [comment]
The travis failure is unrelated (seems like some kind of flake)

**Pull request merged**